Parsers/ASimAuthentication/Tests/Cisco_Meraki_ASimAuthentication_DataTest.csv

```diff
@@ -0,0 +1,11 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 18236 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Meraki""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 18236 records (100.0%) for field [EventVendor] of type [Enumerated]: [""Cisco""] (Schema:Authentication)"
+"(0) Error: 66 invalid value(s) (up to 10 listed) in 18235 records (99.99%) for field [DvcHostname] of type [Hostname]: [""CDSAPGF_08"",""CDSAPGF_05_6_"",""CDSAPGF_02_1_"",""CDSAPGF_03"",""CDSAPGF_07"",""CDSAPF3_01"",""CDSAPF1_02"",""CDSAPF5_11"",""CDSAPF5_07"",""CDSAPF4_11""] (Schema:Authentication)"
```

Comment:
Please resolve this by ensuring we have only the hostname value in this field.

Reply:
As discussed there is an issue in the regex from the data test for some of the values host field contains the underscore and because of that it shows the error.